src/split-panel/__tests__/split-panel.test.tsx

```diff
@@ -71,221 +72,230 @@ afterEach(() => {
 });
 
 describe('Split panel', () => {
-  test('throws error when split panel is used outside its context', () => {
-    expect(() => render(<SplitPanel {...defaultProps} />)).toThrow('Split panel can only be used inside app layout');
-  });
-
-  test('renders panel in bottom position', () => {
-    const { wrapper } = renderSplitPanel({ contextProps: { position: 'bottom' } });
-    expect(wrapper.findOpenPanelBottom()).not.toBeNull();
-    expect(wrapper.findOpenPanelSide()).toBeNull();
-  });
-
-  test('renders panel in side position', () => {
-    const { wrapper } = renderSplitPanel({ contextProps: { position: 'side' } });
-    expect(wrapper.findOpenPanelBottom()).toBeNull();
-    expect(wrapper.findOpenPanelSide()).not.toBeNull();
-  });
-
-  describe.each(['bottom', 'side'] as const)('Position %s', position => {
-    test('displays header, content and aria-label', () => {
-      const { wrapper } = renderSplitPanel({ contextProps: { position } });
-      expect(wrapper.findHeader().getElement()).toHaveTextContent('Split panel header');
-      expect(wrapper.getElement()).toHaveTextContent('Split panel content');
-      expect(wrapper.findSlider()!.getElement()).toHaveAttribute('aria-label', 'resizeHandleAriaLabel');
+  describe.each<SplitPanelProps['closeBehavior']>(['collapse', 'hide'])('closeBehavior=%s', closeBehavior => {
```

Comment:
Wrapped all existing tests in this suite in a `describe.each` to test both values of `closeBehavior`.

Using `testIf` for the cases involving the open split panel button, because it doesn't exist when `closeBehavior="hide"`